x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_actions/actions/component/usage_item.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiBadge,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiLink,
+  EuiText,
+  EuiTextTruncate,
+  EuiIcon,
+} from '@elastic/eui';
+import React from 'react';
+
+import { useKibana } from '../../../../../../hooks/use_kibana';
+import { InferenceUsageInfo } from '../../../../types';
+
+interface UsageProps {
+  usageItem: InferenceUsageInfo;
+}
+export const UsageItem: React.FC<UsageProps> = ({ usageItem }) => {
+  const {
+    services: { application },
+  } = useKibana();
+  const handleNavigateToIndex = () => {
+    if (usageItem.type === 'Index') {
+      application?.navigateToApp('enterprise_search', {
+        path: `content/search_indices/${usageItem.label}`,
```

Comment:
nitpick - nonblocking: these should be in a constants if possible at least partially.